extensions/amp-ima-video/validator-amp-ima-video.protoascii

```diff
@@ -67,3 +67,26 @@ tags: {  # <amp-ima-video>
     supported_layouts: RESPONSIVE
   }
 }
+
+tags: {
+  html_format: AMP
+  tag_name: "TRACK"
+  spec_name: "amp-ima-video > track"
+  mandatory_parent: "AMP-IMA-VIDEO"
+  # <amp-bind>
+  attrs: { name: "[label]" }
```

Comment:
Also, why is "[default]" not here?

Reply:
keeping it same as amp-video and amp-audio. May makes sense to support binding `default` as well. Would require runtime changes as well.